lib/ugrid_checks/check.py

```diff
@@ -284,27 +297,123 @@ def check_attr_mismatch(attr_name):
             check_attr_mismatch("standard_name")
             check_attr_mismatch("units")
 
-            # Do the data-values check.  This is potentially costly.
-            if self.do_data_checks:
-                # TODO: enable data-value checks by attaching lazy data arrays
-                # to scan variables.
-                assert bounds_var.data is not None
-                raise ValueError("Not ready for data-value checks.")
-                log_bounds_statement("A205", "???")
+            if location == "node":
+                msg = (
+                    "has a 'bounds' attribute, which is not valid for a "
+                    "coordinate of 'node' location."
+                )
+                log_bounds_statement("R299", msg)
```

Comment:
This needs a proper code to be assigned

Reply:
See [71f18b](https://github.com/pp-mo/ugrid-checks/pull/45/commits/571f18b1fc98327ed0d121405fc62d76f1aac2ed)